frontend/views.py

```diff
@@ -57,3 +57,11 @@ def get(self, request, *args, **kwargs):
 class CountriesExpensesView(View):
     def get(self, request):
         return render(request, 'frontend/countriesExpenses.html', context=None)
+
+
+class DespesasEReceitas(View, ):
```

Comment:
View names should be in english. Remember to change on other places using this view